src/com/palmergames/bukkit/towny/db/TownySQLSource.java

```diff
@@ -1933,8 +1975,96 @@ public void cleanupBackups() {
     public void deleteUnusedResidents() {
     
     }
-    
-    @SuppressWarnings("deprecation")
+
+	public boolean loadPlotGroupList() {
+		TownyMessaging.sendDebugMsg("Loading PlotGroup List");
+		if (!getContext())
+			return false;
+		try {
+			Statement s = cntx.createStatement();
+			ResultSet rs = s.executeQuery("SELECT groupID,town,groupName FROM " + tb_prefix + "PLOTGROUPS");
+
+			while (rs.next()) {
+				
+				UUID id = UUID.fromString(rs.getString("groupID"));
+				Town town = getTown(rs.getString("town"));
+				String groupName = rs.getString("groupName");
+				
```

Comment:
missing resident, price

Reply:
only missing price.

Reply:
that's not needed during the list loading phase, it's added during the loadPlotGroups() phase.

Reply:
my bad.